packages/ui/src/modal-side/ModalSide.tsx

```diff
@@ -78,9 +78,10 @@ export default function ModalSide({
           <Dialog className="ui-ModalSide--AriaDialog">
             {({ close }) => (
               <>
-                <header className="ui-ModalSide--Header">
-                  <ButtonModalClose label="Close modal" onPress={close} />
-                </header>
+                <ButtonModalClose
```

Comment:
Removing the header element is probably good for accessibility, but then we need to update the styles as this moves the button location.